### PR TITLE
Update stackoverflow-dark.css

### PR DESCRIPTION
**Pull request by Zeeex**
- Fixed new topbar header
- Fixed white herobox when not logged in

### CODE REVIEW

File: stackoverflow-dark.css [hunk 1]
```diff
@@ -88,8 +88,8 @@
 
   /* === background color === */
   /* from darkest to lightest grey */
-  .search-container input[type=text] {
-    background-color: #000 !important;
+  .search-container input[type=text], .topbar.newheader .topbar-wrapper .topbar-links .search-container #search input[name="q"] { 
+    background-color: #222 !important;
```
Comment (Mottie) [inline stackoverflow-dark.css]: Hi @Zeeex!

Thanks for your contribution!

Why change the background color here? Is there a reason why some of these same definitions are repeated in line 174 below?

And why am I not seeing the "newheader" on Stack overflow? Is there an opt-in somewhere to enable it? Thanks!

Comment (Zeeex) [inline stackoverflow-dark.css]: Search looked out of place if it was black.
I'm guessing they are trying out something, or could be that I'm not logged in. (I didn't opt in anywhere)
image

Comment (Mottie) [inline stackoverflow-dark.css]: Would you please move the duplicate definitions down to line 174-ish...

**Comment by Zeeex** [comment]
If I delete those lines, custom rules don't apply for search.
As for the bottom ones I could remove 
>.topbar-wrapper .topbar-links

without breaking anything.

File: stackoverflow-dark.css [hunk 2]
```diff
@@ -163,11 +163,16 @@
...
+#herobox.alternate {
+    background-color: rgba(0, 0, 0, 0.2) !important;
+}
 
   #header #topbar, .revision-comment, .date-span, .post-tag, .form-submit input, .preferred-login,
   .mdhelp, .wmd-help-button.active-help, .date_brick, .more-info, .review-stats-current-user,
-  .review-stats-count-current-user {
-    background: #222 !important;
+  .review-stats-count-current-user, .topbar.newheader, .topbar-menu-links, .network-items>a, .icon-help, .icon-help:hover, a:nth-child(2):hover  {
+    background-color: #222 !important;
```
Comment (Mottie) [inline stackoverflow-dark.css]: Actually, please don't add a `-color` to the background... it is meant to override the background image. You can move whatever definitions that were here up to line 91.

Comment (Zeeex) [inline stackoverflow-dark.css]: When background:
image

When background-color:
image

Sorry about the random search text :P.

**Comment by Zeeex** [comment]
Sorry for the delay, I've fixed the style to not break the current settings. (Thinking the new style was rolled to everyone).

**Comment by Mottie** [comment]
Ok, thanks for all your help!

**Comment by Mottie** [comment]
I ended up combining a few of the definitions, please let me know if it breaks anything!

**Comment by Zeeex** [comment]
It looks good! Thanks for the cleanup.

**Pull request merged**